sdk_extensions/zpages/build.gradle

```diff
@@ -0,0 +1,19 @@
+plugins {
+    id "java"
+    id "maven-publish"
+
+    id "ru.vyarus.animalsniffer"
+}
+
+description = 'OpenTelemetry - zPages'
+ext.moduleName = "io.opentelemetry.sdk.extension.zpages"
+
+dependencies {
+    api project(':opentelemetry-api'),
+            project(':opentelemetry-sdk')
+
+    implementation libraries.guava
+    implementation 'com.sun.net.httpserver:http:20070405'
```

Comment:
Is this referring to the httpserver dependency?

Reply:
Yup - I think you mentioned you needed this to get the IDE to recognize the classes, so compileOnly should work without either requiring users to download it since we're assuming it's already in the JDK anyways or excluding it as a transitive.